src/game/gameTimer.js

```diff
@@ -0,0 +1,48 @@
+import { finishGame } from "./gameResult";
+
+const TIMER = document.querySelector(".timer"); // klasa kontenera w którym znajduje się zegar | do zmiany jeśli będziemy mieli widok
```

Comment:
`docs` or `todo` in English, please, and rather upon the line, you want to comment and not on its right or beneath it.

Reply:
forgot

Reply:
Move it to `line 2` 😄